src/main/scala/com/earldouglas/xwp/ContainerPlugin.scala

```diff
@@ -46,6 +49,7 @@ object ContainerPlugin extends AutoPlugin {
       Seq(libraryDependencies ++= (containerLibs in conf).value.map(_ % conf)) ++
       inConfig(conf)(Seq(
         start              := (startTask dependsOn webappPrepare).value
+      , develop            := (developTask dependsOn webappPrepare).value
```

Comment:
My point was that some application containers(tomcat in particular) will load resources from the webapp first, system classpath second. 

The develop task will be effectively broken in tomcat's default setting, unless you either change the setting for webapp classloader delegation, or create a webappPrepare task that doesn't package the dependencies into the webapp at all.

The later is option is more efficient, and portable.